### ISSUE TITLE
[Suggestion] Remove source code for support legacy Python versions

### ISSUE THREAD

**Issue by thombashi** [trentm/python-markdown2#305]
In the current repository, I can find source codes for a specific Python version that already dropped supporting (Python 2.3, 2.4, etc.).
I assume that these codes are no longer needed.
Could I possibly create a PR to remove these source code?
It will improve source code readability and maintainability.

**Comment by nicholasserra** [comment]
Yes, I think it's fine to remove those. Maybe just keep 2.6 support?

**Comment by thombashi** [comment]
Thank you for your response.
I had make a PR about this issue: #306
In the PR, remain unchanged Python 2.6 / 3.3 support.

Since Python 2.6 and 3.3 is already EOL, these versions also might be okay to drop.
